specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -4019,16 +4647,21 @@
               "$ref": "#/definitions/CloudError"
             }
           }
+        },
+        "x-ms-examples": {
+          "Get a Capacity Reservation.": {
+            "$ref": "./examples/GetACapacityReservation.json"
+          }
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/vmSizes": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/capacityReservationGroups/{capacityReservationGroupName}/capacityreservations": {
       "get": {
         "tags": [
-          "VirtualMachines"
+          "CapacityReservation"
         ],
```

Comment:
im not sure if the differences markers are showing correctly. are you placing VirtualMachines_ListAvailableSizes with CapacityReservations_ListByCapacityReservationGroup?

Reply:
Similar to other comment , the difference markers are not showing changes correctly here.